modules/on-prem/pages/standalone.adoc

```diff
@@ -55,6 +57,8 @@ If you would like to access JupyterLab from a remote browser, type `jupyter lab
 
 JupyterLab only has access to the folder where it is launched. If you need to access files at a different location, launch JupyterLab in that location.
 
+To completely uninstall, remove the folder where you installed the workbench, clean the shell profile (normally the `.bashrc` file in your home directory) if you allowed the installer to initialize your system, and remove the `$HOME/.jupyter/lab/user-settings/@tigergraph` folder.
```

Comment:
Do "remove" and "clean" have different meanings? Suggest changing to "remove" if not

Reply:
yeah, it actually means a different thing than deleting the file. Updated the file.